core/controlplane/config/templates/cloud-config-worker

```diff
@@ -1391,6 +1391,8 @@ write_files:
 
       mkdir -p /etc/ld.so.conf.d/ 2>/dev/null
       echo "/opt/nvidia/current/lib64" > /etc/ld.so.conf.d/nvidia.conf
+      rm /opt/nvidia/current/lib64/libEGL.so.1
```

Comment:
Hi! Thanks for supporting new GPU instance types!! (I'm the original contributor of this feature.)

This `rm` and `ln` would be some workaround for [this bug](https://bugs.launchpad.net/ubuntu/+source/nvidia-graphics-drivers-375/+bug/1662860).

I think referring the bug in comments might help to future contributors because the bug will be fixed in the near future.

What do you think?

Reply:
@everpeace Thanks for the suggestion 👍 Seems like so to me - I will add a comment anyway.
@kylegato Please correct me if I was wrong!

Reply:
Sounds good to me, nice find on that bug @everpeace